crates/bioauth-flow-rpc/src/errors/authenticate.rs

```diff
@@ -4,18 +4,14 @@ use rpc_validator_key_logic::Error as ValidatorKeyError;
 use sp_api::ApiError;
 use sp_runtime::transaction_validity::InvalidTransaction;
 
-use super::{api_error_code, sign::Error as SignError};
+use super::{api_error_code, robonode_request::Error as RobonodeRequestError};
```

Comment:
We have a lot of items to keep track of as is, let's minimize the aliases like `RobonodeRequestError` and use `robonode_request::Error` directly here and everywhere else

Reply:
Improved